models/backbones/mobilenetv2.py

```diff
@@ -0,0 +1,69 @@
+import torch
+import torchvision
+import torchvision.models as models
+from torchvision.models.mobilenetv2 import InvertedResidual
+
+from .abstract_backbone import AbstractBackbone
+
+
+def layer_props(layer):
```

Comment:
please, add docstring like:
def add(a, b):
    """
    Add two numbers and return the result.

    Args:
        a (int): The first number.
        b (int): The second number.

    Returns:
        int: The sum of the two numbers.
    """
    return a + b

or any other docstring format